app/src/main/java/com/adiliqbl/wallpapers/ui/images/ImageDetailsFragment.kt

```diff
@@ -14,17 +21,38 @@ class ImageDetailsFragment : BaseFragment<ImageDetailsViewModel>() {
 
     companion object {
         private const val KEY_IMAGE = "image"
+        private const val KEY_TRANSITION_NAME = "transitionName"
 
         @JvmStatic
```

Comment:
@adiliqbl JvmStatic should only be used if you are sure that your code will be accessed from a Java file